testcases/master/template-kvm-unit-tests.yaml.jinja2

```diff
@@ -24,4 +24,5 @@
       parameters:
         SKIP_INSTALL: 'true'
         SMP: {% if DEVICE_TYPE == "juno-r2" %}'false'{% else %}'true'{% endif %}
+        GIT_REF: "{{KVM_UNIT_TESTS_REVISION | default('perkele_need_to_specify_something')}}"
```

Comment:
While I like the default a lot, maybe it should just be 'master' ?

Reply:
Maybe we should use this as a default for all the GIT_REF's and/or branches we try to checkout so the user have to do something active in order to get the sha/branch he/she wants?

Reply:
We want defaults to work, not to fail.

Reply:
@roxell do not force the user for it.

Reply:
@roxell default as master is not harmful.

Reply:
I get it, when I want more comments on my patches I should start using swear words :)

Reply:
yes =)